Userland/Libraries/LibJS/Runtime/Temporal/TimeZone.cpp

```diff
@@ -435,14 +415,43 @@ ThrowCompletionOr<Object*> to_temporal_time_zone(GlobalObject& global_object, Va
     // 2. Let identifier be ? ToString(temporalTimeZoneLike).
     auto identifier = TRY(temporal_time_zone_like.to_string(global_object));
 
-    // 3. Let result be ? ParseTemporalTimeZone(identifier).
-    auto result = TRY(parse_temporal_time_zone(global_object, identifier));
+    // 3. Let parseResult be ? ParseTemporalTimeZoneString(identifier).
+    auto parse_result = TRY(parse_temporal_time_zone_string(global_object, identifier));
+
+    // TODO: This currently cannot be tested as ParseTemporalTimeZoneString only considers
+    //       TimeZoneIANAName for the returned [[Name]] slot, not TimeZoneUTCOffsetName.
+    //       So when we provide a numeric time zone offset, this branch won't be executed,
+    //       and if we provide an IANA name, it won't be a valid TimeZoneNumericUTCOffset.
+    //       This should be fixed by https://github.com/tc39/proposal-temporal/pull/1941.
```

Comment:
incredibly extreme nitpick: the `.` at the end breaks this link on github :^)

Reply:
lol, fixed